demisto_sdk/commands/common/tools.py

```diff
@@ -2006,13 +2020,13 @@ def get_content_path() -> Union[str, PathLike, None]:
 
         if not is_fork_repo and not is_external_repo:
             raise git.InvalidGitRepositoryError
-        return git_repo.working_dir
+        return Path(git_repo.working_dir).absolute()  # type: ignore[arg-type]
     except (git.InvalidGitRepositoryError, git.NoSuchPathError):
         if not os.getenv("DEMISTO_SDK_IGNORE_CONTENT_WARNING"):
             logger.info(
                 "[yellow]Please run demisto-sdk in content repository![/yellow]"
             )
-    return ""
+    return Path.cwd()
```

Comment:
thank you

Reply:
This make issues in tests (where we use `ChangeCWD`). Changed :(

Reply:
oof